code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -1647,6 +1647,22 @@
 	density = 4.33
 	specheatcap = 2.64
 
+/datum/reagent/nitroglycerin/on_mob_life(var/mob/living/M)
+	M.adjustToxLoss(2 * REM)
+	if(prob(80))
+		M.adjustOxyLoss(2 * REM)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		var/datum/organ/internal/heart/E = H.internal_organs_by_name["heart"]
+		if(istype(E) && !E.robotic)
+			if(E.damage > 0)
+				E.damage = max(0, E.damage - 0.2)
+		if(prob(50))
+			H.custom_pain("You feel a throbbing pain in your head", 0)
```

Comment:
did you test this? because in practice I do believe this means you will get spammed with this message about once every four seconds, which feels bad

Reply:
that's about pure nitroglycerin which is extremely rare(there is almost no way to get it without explosion) and way stronger because it's undiluted, so it felt fitting to cause constant suffering

Reply:
constant suffering is fine, constant message spam is just annoying

Reply:
it's a part of constant suffering, these pain messages cloud the text window and make it harder to read/understand conversations like a real strong headache
just really makes you regret eating floorpills

Reply:
Spamming the chatlog just seems a bit meta for 'suffering'. If you want it to take up more space or be obvious use a larger font size.

Reply:
sure, sounds reasonable